ons-spark/spark-concepts/partitions.md

```diff
@@ -391,9 +391,9 @@ Number of partitions in rescue_by_area DataFrame:	 200
 1 SE5                 42
 2 SE17                20
 3 E7                  45
-4 SE13                42
-5 N6                   9
-[1] "Number of partitions in count_by_area DataFrame:16"
+4 N6                   9
+5 DA15                12
+[1] "Number of partitions in count_by_area DataFrame:32"
```

Comment:
The default number was thought to be 16 not 32. Reason for this change is unknown